acceptancetests/deploy_stack.py

```diff
@@ -95,6 +96,15 @@
     source: /dev/null
     type: disk
 """
+IAAS_LXD_PROFILE = """
```

Comment:
do we need this for the iaas model? I don't see what's in the bundle here?

Reply:
Hmm, so in looking at this since the bundle is provided by the calling into this maybe the lxd profile bits should be passed in vs hard coded as well. Ok for this PR but a general question for the future.

Reply:
Yes, good shout! I'm definitely not sold on this bit.